modules/rosa-policy-identity-access-management.adoc

```diff
@@ -36,7 +36,7 @@ Each of these access types have different levels of access to components:
 
 | OpenShift Cluster Manager (OCM) | R/W | No access | No access | No access
 | OpenShift console | No access | R/W | R/W | No access
-| Node Operatiing  system | No access | A specific list of elevated OS and network permissions. | A specific list of elevated OS and network permissions. | No access
+| Node Operating system | No access | A specific list of elevated OS and network permissions. | A specific list of elevated OS and network permissions. | No access
```

Comment:
@arendej thank you for fixing this typo. Can you please change "Operating" to "operating", lower case? Thanks!